recipes/slda/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "slda" %}
+{% set version = "0.1.6" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://github.com/Savvysherpa/slda/archive/{{ version }}.tar.gz
+    sha256: aeabfe4edefc566ee97c107f8f9b2c6a478595e24fb35d44d9eeaaa876fb90ce
+
+build:
+    number: 0
+    skip: True  # [win or not py3k]
+
+requirements:
+    build:
+        - cython
+        - cythongsl
```

Comment:
The problem with this is @mmb90 and @ocefpaf `slda` is actually [linking]( https://travis-ci.org/conda-forge/staged-recipes/jobs/146218765#L1129 ) to `gsl` unlike `cythongsl`. As a result, if `gsl` gets updated and we don't have a way to properly pin it here (e.g. `gsl`) is not listed, this will end up being broken. As this has been merged in spite of this comment, I would please ask you to fix it in the feedstock.